caddyhttp/proxy/upstream.go

```diff
@@ -150,7 +150,8 @@ func (u *staticUpstream) From() string {
 
 func (u *staticUpstream) NewHost(host string) (*UpstreamHost, error) {
 	if !strings.HasPrefix(host, "http") &&
-		!strings.HasPrefix(host, "unix:") {
+		!strings.HasPrefix(host, "unix:") &&
+		!strings.HasPrefix(host, "quic:") {
```

Comment:
Okay, so just to be sure I'm clear, using `quic:` in the Caddyfile this way means that the upstream server _has to use QUIC each and every time_. That's acceptable, I just want to be sure that's clear for the documentation. It does not only enable QUIC to upstream, it _forces_ it.